engine/src/mac-dialog.mm

```diff
@@ -219,14 +219,28 @@ void MCPlatformBeginFolderDialog(MCPlatformWindowRef p_owner, MCStringRef p_titl
         [t_panel setMessage: [NSString stringWithMCStringRef: p_prompt]];
     }
     else
-        [t_panel setTitle: [NSString stringWithMCStringRef: p_prompt]];
+    {
+        extern uint4 MCmajorosversion;
+        if (MCmajorosversion >= 0x10B0)
+            [t_panel setMessage: [NSString stringWithMCStringRef: p_prompt]];
+        else
+            [t_panel setTitle: [NSString stringWithMCStringRef: p_prompt]];
+        
+    }
+        //[t_panel setTitle: [NSString stringWithMCStringRef: p_prompt]];
```

Comment:
Also, we should update the docs to explicitly explain this situation - and make sure it is clear that you should use 'prompt' for any critical information, as 'title' may or may not appear, depending on platform.